lua51/PSPBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=lua51
```

Comment:
Is there a reason this pkgname is not just lua?

Reply:
the lua package already exist and is lua5.2 no ? 
On arch they package the latest lua as `lua`and lua5.x as `lua5x`

Reply:
That makes sense. Thanks for clarifying!